app/controllers/api/databases.php

```diff
@@ -2781,3 +2879,118 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
+        $cursor = reset($cursor);
+        if ($cursor) {
+            $documentId = $cursor->getValue();
+            $cursorDocument = Authorization::skip(fn()=>$dbForProject->getDocument('slowQueries', $documentId));
+            if ($cursorDocument->isEmpty()) {
+                throw new Exception(Exception::GENERAL_CURSOR_NOT_FOUND, "Slow query '{$documentId}' for the 'cursor' value not found.");
+            }
+            $cursor->setValue($cursorDocument);
+        }
+
+        $response->dynamic(new Document([
+            'documents' => Authorization::skip(fn()=>$dbForProject->find('slowQueries', $queries)),
+            'total' => Authorization::skip(fn()=>$dbForProject->count('slowQueries', Query::groupByType($queries)['filters'], APP_LIMIT_COUNT)),
+        ]), Response::MODEL_DOCUMENT_LIST);
+    });
+
+App::get('/v1/databases/slow-queries/:documentId')
+    ->desc('Get Slow Query Document')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('sdk.hide', true)
+    ->label('scope', 'documents.read')
+    ->label('usage.metric', 'documents.{scope}.requests.read')
+   // ->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.method', 'getSlowQuery')
+    ->label('sdk.description', '/docs/references/databases/get-document.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_DOCUMENT)
+    ->param('documentId', '', new UID(), 'Document ID.')
+    ->inject('response')
+    ->inject('dbForProject')
+    ->inject('mode')
+    ->inject('project')
+    ->action(function (string $documentId, Response $response, Database $dbForProject) {
+        $document = Authorization::skip(fn () => $dbForProject->getDocument('slowQueries', $documentId));
+        if ($document->isEmpty()) {
+            throw new Exception(Exception::DOCUMENT_NOT_FOUND);
+        }
+
+        $response->dynamic($document, Response::MODEL_DOCUMENT);
+    });
+
+App::delete('/v1/databases/slow-queries/:documentId')
+    ->desc('Delete Slow query Document')
+    ->desc('List  Documents')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('sdk.hide', true)
+    ->label('scope', 'documents.write')
+    ->label('usage.metric', 'documents.{scope}.requests.delete')
+    //->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('event', 'slowQueries.documents.[documentId].delete')
+    ->label('audits.event', 'document.delete')
+    ->label('audits.resource', 'database/{request.databaseId}/collection/{request.collectionId}/document/{request.documentId}')
```

Comment:
We can remove the `event`, and `audit.*` labels

Reply:
Done!